src/linker/Linker/XmlDependencyRecorder.cs

```diff
@@ -173,7 +179,7 @@ bool WillAssemblyBeModified (AssemblyDefinition assembly)
 			}
 		}
 
-		bool ShouldRecord (object o)
+		bool ShouldRecord (object? o)
```

Comment:
Would it make sense to return false if `o` is null?

Reply:
It might, but `TokenString` supports logging edges with null source and I'd rather not change that here.